### PR TITLE
fix: display Operate link without trailing slash

### LINKED ISSUE #3596: Unable to open process in Operate after deployment
### Describe the bug

Operate answers with 
```
Whitelabel Error Page

This application has no explicit mapping for /error, so you are seeing this as a fallback.
```
When I click on the link after deploying for the first time:

grafik

### Steps to reproduce

1. create a new process model
2. make it executable
3. deploy it to a fresh SaaS cluster
4. Click on the link from the Popup.

### Expected behavior

Operate opens the deployed process instead of an error page.

The link provided from the modeler is:

```
https://bru-2.operate.camunda.io/7c1184c4-040c-4d11-af4e-ef08888fe6e6/instances/?process=PaymentProcess&version=all&active=true&incidents=true
```

It should be (no trailing `/` in path name part):

```
https://bru-2.operate.camunda.io/7c1184c4-040c-4d11-af4e-ef08888fe6e6/instances?process=PaymentProcess&version=1&active=true&incidents=true
```

### Environment

**Camunda Modeler system information**
 * Version: 5.10.0
 * Operating System: Windows 10 amd64
 * Plugins: Cloud Connect, bpmn-js Token Simulation, Migration, Transaction Boundaries, Camunda Property Infos, Tooltip Plugin
 * Execution Platform: BPMN - Camunda Platform 8

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by barmac**
Closes #3596

### CODE REVIEW

File: client/src/plugins/zeebe-plugin/start-instance-plugin/__tests__/StartInstancePluginSpec.js [hunk 1]
```diff
@@ -409,7 +409,7 @@ describe('<StartInstancePlugin> (Zeebe)', () => {
       const notificationHTML = shallow(notification.content).html().replace(/&amp;/g, '&');
 
       expect(notificationHTML).to.include(
-        'https://REGION.operate.camunda.io/CLUSTER_ID/instances/undefined'
+        'https://region.operate.camunda.io/CLUSTER_ID/instances/test'
```
Comment (nikku) [inline client/src/plugins/zeebe-plugin/start-instance-plugin/__tests__/StartInstancePluginSpec.js:412]: Guess this is automatically lower-cased due to using `URL` now?

Comment (nikku) [reply client/src/plugins/zeebe-plugin/start-instance-plugin/__tests__/StartInstancePluginSpec.js:412]: I'd propose to also change the input (region) in the process.

Comment (barmac) [reply client/src/plugins/zeebe-plugin/start-instance-plugin/__tests__/StartInstancePluginSpec.js:412]: Yup, that's how URL works. I will adjust the test data then.

File: client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js [hunk 2]
```diff
@@ -612,27 +617,25 @@ function CloudLink(props) {
...
     return null;
   }
 
-  const query = `?process=${processId}&version=all&active=true&incidents=true`;
-  const cloudUrl = `https://${camundaCloudClusterRegion}.operate.camunda.io/${camundaCloudClusterId}/instances/${query}`;
+  const cloudUrl = getCloudLink(endpoint, response);
+  cloudUrl.searchParams.set('process', processId);
+  cloudUrl.searchParams.set('version', 'all');
+  cloudUrl.searchParams.set('active', 'true');
+  cloudUrl.searchParams.set('incidents', 'true');
```
Comment (nikku) [inline client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js:630]: Nice stuff!

**Review by nikku — APPROVED** [review]

**Review by nikku — APPROVED** [review]

File: client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js (hunk 2 above)
Comment (barmac) [reply client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js:630]: Yeah I think it's safer than manual typing the queries.

**Pull request merged**